src/paperless_tesseract/parsers.py

```diff
@@ -66,6 +66,10 @@ def is_image(self, mime_type):
             "image/gif",
         ]
 
+    def has_alpha(self, image):
+        with Image.open(image) as im:
+            return im.mode in ('RGBA', 'LA')
+
```

Comment:
Can opening the image fail here? Same for below.

Reply:
Good question. I had an exception handler in place first, but that degraded the coverage report on the "original" fork and I couldn't think of an test case. I'm not familiar with the code base, but if `get_dpi(input_file)` and `calculate_a4_dpi(input_file)` in the lines before do not fail, I see no reason why the call to `has_alpha(input_file)` should fail (same for the code below).